fbgemm_gpu/codegen/training/backward/embedding_backward_split_host_template.cpp

```diff
@@ -1191,7 +1199,8 @@ TORCH_LIBRARY_FRAGMENT({{ lib_name }}, m) {
           {%- if ssd %}
           "    Tensor[]? ssd_tensors=None,"
           {%- endif %}
-          "   float gwd_lower_bound=0 "
+          "   float gwd_lower_bound=0, "
+          "   bool mixed_D=False"
```

Comment:
Hi @q10. Consider setting default **mixed_D** value to **True**.

By splitting backend and frontend into 2 separate entities, you force optimized kernel launch with mode that is not currently supported (mixed_D in this case). By setting default value to True, upstream backward kernel will be launched in all the cases, so without frontend changes this PR's functionality won't be covered with unit-tests.